sys/net/gnrc/network_layer/ipv6/ext/frag/gnrc_ipv6_ext_frag.c

```diff
@@ -255,7 +255,12 @@ void gnrc_ipv6_ext_frag_send(gnrc_ipv6_ext_frag_send_t *snd_buf)
     /* tell gnrc_ipv6 to send the above prepared fragment */
     msg.type = GNRC_IPV6_EXT_FRAG_SEND;
     msg.content.ptr = to_send;
-    msg_try_send(&msg, gnrc_ipv6_pid);
+    if (msg_try_send(&msg, gnrc_ipv6_pid) <= 0) {
+        DEBUG("ipv6_ext_frag: Unable to send fragment, canceling fragmentation\n");
+        gnrc_pktbuf_release(to_send);
+        _snd_buf_free(snd_buf);
+        return;
```

Comment:
can we propagate that error up the stack?

Reply:
Of course!

Reply:
Done! Made it a separate commit, so the other cases of `_snd_buf_free()` are included as well.